x/upgrade/types/plan.go

```diff
@@ -73,16 +67,5 @@ func (p Plan) DueAt() string {
 
 // IsIBCPlan will return true if plan includes IBC client information
 func (p Plan) IsIBCPlan() bool {
-	return p.UpgradedClientState != nil
-}
-
-// UnpackInterfaces implements UnpackInterfacesMessage.UnpackInterfaces
-func (p Plan) UnpackInterfaces(unpacker codectypes.AnyUnpacker) error {
```

Comment:
One last question @amaurym. Reserving a field number doesn't use the previous type (`Any`), so this function should be safe to remove?

Reply:
Yes!